cluster-provision/k8s/check-cluster-up.sh

```diff
@@ -53,8 +68,16 @@ export KUBEVIRTCI_GOCLI_CONTAINER=quay.io/kubevirtci/gocli:latest
 
     trap cleanup EXIT ERR SIGINT SIGTERM SIGQUIT
     bash -x ./cluster-up/up.sh
-    timeout 210s bash -c "until ${ksh} wait --for=condition=Ready pod --timeout=30s --all -l app!=whereabouts; do sleep 1; done"
-    timeout 210s bash -c "until ${ksh} wait --for=condition=Ready pod --timeout=30s -n kube-system --all -l app!=whereabouts; do sleep 1; done"
+
+    namespaces=$(${ksh} get namespaces -o jsonpath='{.items[*].metadata.name}')
```

Comment:
sorry for being unclear, @brianmcarey - I actually meant moving the ns gathering into the gather_cluster_state function. This way we might use it like this:
```suggestion
    if ! timeout 210s bash -c "until ${ksh} wait --for=condition=Ready pod --timeout=30s --all -l app!=whereabouts; do sleep 1; done" || timeout 210s bash -c "until ${ksh} wait --for=condition=Ready pod --timeout=30s -n kube-system --all -l app!=whereabouts; do sleep 1; done"; then
      gather_cluster_state
      exit 1
    fi
```